airflow-core/src/airflow/api_fastapi/execution_api/routes/connections.py

```diff
@@ -23,8 +23,8 @@
 
 from airflow.api_fastapi.execution_api.datamodels.connection import ConnectionResponse
 from airflow.api_fastapi.execution_api.deps import JWTBearerDep
-from airflow.exceptions import AirflowNotFoundException
 from airflow.models.connection import Connection
+from airflow.sdk.exceptions import AirflowNotFoundException
```

Comment:
Same as above: We shouldn't import exception from Task SDK in core and vice-versa